services/worker/tests/job_runners/split/test_opt_in_out_urls_scan_from_streaming.py

```diff
@@ -191,10 +188,22 @@ def test_compute(
         kind=job_runner.processing_step.cache_kind, dataset=dataset, config=config, split=split
     )
     assert cached_response
-    assert cached_response["content"] == expected_content
     assert cached_response["http_status"] == HTTPStatus.OK
     assert cached_response["error_code"] is None
 
+    content = cached_response["content"]
+    assert content
+    assert content["has_urls_columns"] == expected_content["has_urls_columns"]
+    assert content["num_scanned_rows"] == expected_content["num_scanned_rows"]
+    assert content["urls_columns"] == expected_content["urls_columns"]
+    assert content["num_opt_out_urls"] == expected_content["num_opt_out_urls"]
+    assert content["num_opt_in_urls"] == expected_content["num_opt_in_urls"]
+    assert content["num_urls"] == expected_content["num_urls"]
+    if expected_content["has_urls_columns"]:
+        assert content["opt_in_out_urls_source"] is not None and isinstance(content["opt_in_out_urls_source"], dict)
+    else:
+        assert content["opt_in_out_urls_source"] is None
+
```

Comment:
We might want to also test that the contents of the file are what we expect (ie. read the CSV and check the format and values)